gradle.properties

```diff
@@ -14,6 +14,6 @@
 # limitations under the License.
 #
 
-version=1.3.0
+version=1.3.1
```

Comment:
Is this a bug fix or a minor update?, semantic version says you should release under 1.4.0 but you already merged this PR.